subxt/src/blocks/blocks_client.rs

```diff
@@ -48,6 +48,36 @@ where
     /// runtime upgrade. You can attempt to retrieve older blocks,
     /// but may run into errors attempting to work with them.
     pub fn at(
+        &self,
```

Comment:
We could also change the documentation of this function to state `/// Obtain block details given the block hash.`